src/main/webapp/CountryForm.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="UTF-8" />
+<title>Event Form</title>
+<link rel="stylesheet" href="style.css" />
+<div id="content">
+  <div id="location-container"></div>
+  <form action="/api/countries" method="post">
+    <h1>Enter a Country</h1>
+    <p>ID:</p>
+    <textarea name="id" placeholder="1"></textarea>
+
+    <p>Name:</p>
+    <textarea name="name" placeholder="South Korea"></textarea>
+
+    <p>Culture Do's(seperate with ,):</p>
+    <textarea name="cultureDos" placeholder="A,B,C"></textarea>
```

Comment:
Instead of separating by commas you could have multiple `<input>` elements with the same `name` (see this [Stackoverflow answer](https://stackoverflow.com/a/2906814) to see what I mean.

The HTML form could dynamically add more input elements to add more dos/dont fields (but that can be a separate change)